### ISSUE TITLE
Progress stepper column with only title is not aligned on last item

### ISSUE THREAD

**Issue by HenriqueLimas** [eBay/skin#1441]
When used progress stepper as column type and with only title, the last step is not aligned with the rest of the steps.

<img width="243" alt="Screen Shot 2021-05-13 at 9 39 24 AM" src="https://user-images.githubusercontent.com/2222191/118190880-9a7bd480-b3f8-11eb-8e35-047c93066928.png">

I suppose its because of the rule used for `row` where the last step is aligned with flex-end:

```css
.progress-stepper__item:last-child .progress-stepper__text {
    align-self: flex-end;
    margin-right: 4px;
    text-align: right;
}
```

**Comment by agliga** [comment]
I don't believe this is a valid usecase of vertical stepper, it should have text inbetween from what I know. Horizontal should support not having extra text.

**Comment by agliga** [comment]
Fix released in `12.5.0`
